frontend/customization/config/map.json

```diff
@@ -2,8 +2,27 @@
   "searchMapCenter": [44.8, 6.3],
   "maximumZoomLevel": 17,
   "searchMapZoom": 10,
-  "mapCredits": "OpenTopoMap",
-  "mapClassicLayerUrl": "https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png",
-  "mapSatelliteLayerUrl": "https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}",
+  "mapClassicLayers": [
```

Comment:
Je ne vois comment on peut faire ça : Ability to display different layers depending on the zoom
On pourrait mettre un exemple plus complet avec ce cas vu que c'est un fichier d'exemple qui n'est pas utilisé par l'application ?
Par exemple mettre OTM à large échelle et IGN PLAN V2 (https://wxs.ign.fr/cartes/geoportail/wmts?&REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE=normal&TILEMATRIXSET=PM&FORMAT=image/png&LAYER=GEOGRAPHICALGRIDSYSTEMS.PLANIGNV2&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}) du zoom 15 au 18 pour l'exemple ?

Reply:
Voir https://github.com/GeotrekCE/Geotrek-rando-v3/pull/720

Reply:
Configuration added